orttraining/orttraining/python/training/ortmodule/ortmodule.py

```diff
@@ -65,15 +65,32 @@ def forward(self, *inputs, **kwargs):
         '''Dummy documentation for forward method'''
         ...
 
+    def _apply(self, fn):
+        """Override original method to delegate execution to the base module"""
+
+        # Delegation must happen to _flattened_module since methods depend on
+        # _apply to recursively apply the internal setting changes
+        self._flattened_module._apply(fn)
+        return self
+
     def _is_training(self):
-        return self._flattened_module.training and torch.is_grad_enabled()
+        return self.training and torch.is_grad_enabled()
+
+    def train(self: T, mode: bool = True) -> T:
+        """Override original method to delegate execution to the base module"""
```

Comment:
All overriden APIs have "base module" in their docstring. I believe this is an outdated comment when we had a base_module attribute inside Flattened Module

How about using another term such as "original user PyTorch model/module" or something like that? base module reminds some sort of inheritance, which doesnt happen